packages/eslint-plugin-turbo/lib/utils/calculate-inputs.ts

```diff
@@ -132,17 +131,20 @@ function processDotEnv(
 
 function processGlobal(
   workspacePath: string,
-  rootTurboJson: RootSchema | RootSchemaV1
+  schemaV1: RootSchemaV1
 ): EnvironmentConfig {
   return {
-    legacyConfig: processLegacyConfig(rootTurboJson.globalDependencies),
-    env: processEnv(rootTurboJson.globalEnv),
-    passThroughEnv: processPassThroughEnv(rootTurboJson.globalPassThroughEnv),
-    dotEnv: processDotEnv(workspacePath, rootTurboJson.globalDotEnv),
+    legacyConfig: processLegacyConfig(schemaV1.globalDependencies),
+    env: processEnv(schemaV1.globalEnv),
+    passThroughEnv: processPassThroughEnv(schemaV1.globalPassThroughEnv),
+    dotEnv: processDotEnv(workspacePath, schemaV1.globalDotEnv),
```

Comment:
This feels a _little_ confusing since this should work with the V2 schema as well, but the types look explicit to V1 only

Reply:
my understanding is that it can't work with the v2 types, since there are values here that are not present in v2, for example `globalDotEnv`.  what am I missing?

Reply:
Yea, right now v2 is a subset of v1, which is why this is working fine. It's just a little confusing to see the type as v1 explicitly even though that will also work with v2.